layouts/_default/baseof.html

```diff
@@ -4,6 +4,12 @@
     {{ partial "head.html" . }}
   </head>
   <body class="td-{{ .Kind }}">
+    {{ if hugo.IsProduction }}
```

Comment:
From this line, I expected that the lines L9-L10 would not appear in Netlify preview (because it's not prod),
but in the source of the [preview](https://deploy-preview-2801--cncfglossary.netlify.app), those lines are rendered.

My questions are:
1. Is this right? Does Hugo presume Netlify previews as prod?
   (Or, it's just Hugo cannot tell Netlify previews and the real prod (https://glossary.cncf.io) apart?)
2. Would be there no problem if GTM tags are inserted in Netlify preview sites?

Reply:
I think it's not correct. Previews should not have analytics and should not be `true` for `IsProduction`.  It's not a huge concern, however, as it has been how this site has been working from the beginning with Google Analytics and GTM should not be any different.

I suggest we split this problem out as an independent issue and not allow it to hold up merging this PR?

Reply:
Here's [a new issue for this problem](https://github.com/cncf/cncf.io/issues/824).